packages/backend-test-utils/src/database/TestDatabases.test.ts

```diff
@@ -56,81 +56,160 @@ describe('TestDatabases', () => {
     );
   });
 
-  itIfDocker('obeys a provided connection string for postgres 13', async () => {
+  describe('each connect', () => {
```

Comment:
Note: I could not get these (existing) tests to pass locally because I was getting a `Hooks cannot be defined inside tests.` error.  ...Turns out `TestDatabases.create()` introduces an `afterAll()`.

So I moved these to a new describe block and brought the `TestDatabases.create()` call to the top of the block.